CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ set(CMAKE_USER_MAKE_RULES_OVERRIDE "${CMAKE_SOURCE_DIR}/cmake/Platform.cmake")
 
 project (Elektra)
 
-#fix OS X RPATH issues
+#fix macOS RPATH issues
```

Comment:
I do not think so, since I assume that most Mac users already use macOS 10.12 or macOS 10.11.